get-started/index.md

```diff
@@ -135,7 +143,7 @@ to determine that this is a compatible Java application that it should monitor.
 ### Open the Cryostat Web UI
 Let's visit the Cryostat web dashboard UI.
 
-We can get there from the Cryostat resource's Status field:
+We can get there from the `Cryostat` `CR`'s status fields:
 
 {% include howto_step.html
   details-attributes="open"
```

Comment:
In this section, we mention `Topology View`, which probably should be `OpenShift Topology View` to avoid confusion with our `Cryostat Topology View`.